monitoring/bin/create_elasticsearch_datasource_tenant.sh

```diff
@@ -127,7 +129,11 @@ fi
 
 # Check to see if tenant has been deployed in logging.
 log_info "Checking that $tenant has been onboarded to Elasticsearch in the $LOG_NS namespace ..."
-if [[ $(kibana_tenant_exists $tenant) == 1 ]]; then
+
+get_sec_api_url
+get_credentials_from_secret admin
+
+if ! kibana_tenant_exists "${tenantNS}_${tenant}"; then
   log_error "Unable to configure Elasticsearch data source.  The $tenant tenant has not been onboarded.";
```

Comment:
Probably should include the "namespace" in the error message as well.  e.g.  _The tenant [foo] within the namespace [bar] has not been onboarded._